static/src/stylesheets/module/survey/_weekend-reading.scss

```diff
@@ -1,152 +1,159 @@
+#survey-laboratory .illustration-container {
+    min-height: 20rem;
+    height: 200px;
+    width: 100%;
+    position: relative;
+    z-index: 80;
+
+    @include mq($until: tablet) {
```

Comment:
Generally the convention is to put the mobile styles as the defaults and override with media queries for larger widths, i.e.

```css
#survey-laboratory .illustration-container {
    min-height: 10rem;
    height: 200px;
    width: 100%;
    position: relative;
    z-index: 80;

    @include mq(tablet) {
        min-height: 20rem;
    }
}
```
`until` media queries should be reserved for cases where it's really awkward to do it the other way around